### PR TITLE
move root views new file

### PR DESCRIPTION
**Pull request by wwitzel3**
more views pruning

### CODE REVIEW

File: awx/api/views/root.py
Comment (AlanCoding) [inline awx/api/views/root.py:166]: optimization:

`for instance_group in InstanceGroup.objects.prefetch_related('instances').all():`

Comment (wwitzel3) [reply awx/api/views/root.py:166]: Nope. We are moving things first. I'll approve a PR to optimize this after this lands.

**Review by ryanpetrello — APPROVED** [review]

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**
